packages/editor/src/client/blocks/richTextBlock/toolbar/tableMenu.tsx

```diff
@@ -67,50 +91,34 @@ export function TableMenu() {
     })
   }
 
-  const setTableCellBorderColor = (color: string) => {
-    const {selection} = editor
-    if (selection) {
-      const nodes = Array.from(
-        WepublishEditor.nodes(editor, {
-          at: selection,
-          match: node => node.type === BlockFormat.Table
-        })
-      )
-      Transforms.setNodes(
-        editor,
-        {borderColor: color},
-        {at: nodes[0][1], match: node => node.type === BlockFormat.TableCell}
-      )
-    }
-  }
-
   const tableModifyControls = (
     <>
       {!showRemoveConfirm ? (
         <>
-          {isBorderVisible() ? (
-            <Button appearance="subtle" onClick={() => setTableCellBorderColor('transparent')}>
-              {t('blocks.richTextTable.hideBorders')}
-            </Button>
+          {borderColor && borderColor !== 'transparent' ? (
+            <ControlsContainer dividerBottom>
+              <ColorPicker
+                withColor={color => {
+                  setBorderColor(color)
+                }}
+                currentColor={borderColor}
+                label={t('blocks.richTextTable.border')}
+              />
+              <button className="icon-button" onClick={() => setBorderColor('transparent')}>
+                <Icon icon="ban" style={{color: 'red'}} />
```

Comment:
somewhere further above you've changed 'black' to HEX '000000'. In order to by consistent you might as well chose to set a HEX value for red? It's #FF0000  and for 'transparent' you could use #00000000 (looks weird but it's black with full transparency). But in the end I guess it's up to you what you use, maybe just use consistently the same way :)

Reply:
Agree. Personally I like using names as 'black' or 'red'. Setting the <input value={'black'} ... > caused a console warning, saying something like `expected '#xxyycmzz' string ... ` therefore I switched to hex. Therefore I would use names where possible still. If you prefer consistency I will change?

Reply:
I don't really have a strong opinion on this, maybe @roosnic1 knows better if it matters what we use. But I just did a little research and I came upon this: [Here](http://thenewcode.com/781/Which-CSS-Color-System-To-Use-Where) they advise against using color definitions like `black` and use HEX or other systems instead.

Reply:
Changed it, but now it is more inconsistent with rest of the codebase. Think mainly keyword colors are used in the codebase. And they used 'transparent' everywhere. As mentioned I used '#0000000' for black to avoid a console warning on the html basic input element, as an exception. Maybe there would be better ways getting around this.